src/pages/home/report/ReportActionsList.tsx

```diff
@@ -576,6 +577,7 @@ function ReportActionsList({
         ({item: reportAction, index}: ListRenderItemInfo<OnyxTypes.ReportAction>) => {
             return (
                 <ReportActionsListItemRenderer
+                    allReports={allReports}
```

Comment:
@adhorodyski @TMisiukiewicz Isn't passing `allReports` to all report action children will increase the memory usage?